wrphttp/requestResponse.go

```diff
@@ -105,16 +139,38 @@ type entityResponseWriter struct {
 	f wrp.Format
 }
 
-func (erw *entityResponseWriter) WriteWRP(v interface{}) (int, error) {
-	var (
-		output  []byte
-		encoder = wrp.NewEncoderBytes(&output, erw.f)
-	)
+func (erw *entityResponseWriter) WriteWRP(e *Entity) (int, error) {
+	if e == nil {
```

Comment:
Generally, you don't need to do this.  It's ok to allow panics on nil.  That indicates a bug in the calling code, which you don't want to mask with a "normal" call return.